try {
    console.log("hello!");
    // zzz;
    console.log("hello2!");

} catch (error) {
    console.log("catch");

} finally {
    console.log("finally");

}